go/vt/discovery/tablet_picker.go

```diff
@@ -157,12 +157,17 @@ func (tp *TabletPicker) getMatchingTablets(ctx context.Context) []*topo.TabletIn
 			// non-blocking read so that this is fast
 			shortCtx, cancel := context.WithTimeout(ctx, *topo.RemoteOperationTimeout)
 			defer cancel()
-			alias, err := tp.ts.GetCellsAlias(shortCtx, cell, false)
+			_, err := tp.ts.GetCellInfo(ctx, cell, false)
```

Comment:
do we want to use the shortCtx context here too?

Reply:
good catch. I'll fix it.